tests/generated_data/generate_md.py

```diff
@@ -97,23 +96,15 @@ def generate_all_files(
         verify: Whether to verify the newly generated files with the
             local staored.
     """
-    root = Root(1, SPEC_VERSION, EXPIRY, {}, ROLES, True)
-    root.add_key("root", keys["ed25519_0"])
-    root.add_key("timestamp", keys["ed25519_1"])
-    root.add_key("snapshot", keys["ed25519_2"])
-    root.add_key("targets", keys["ed25519_3"])
-
-    md_root: Metadata[Root] = Metadata(root, {})
-
-    timestamp = Timestamp(1, SPEC_VERSION, EXPIRY, MetaFile(1))
-    md_timestamp: Metadata[Timestamp] = Metadata(timestamp, {})
-
-    meta: Dict[str, MetaFile] = {"targets.json": MetaFile(1)}
-    snapshot = Snapshot(1, SPEC_VERSION, EXPIRY, meta)
-    md_snapshot: Metadata[Snapshot] = Metadata(snapshot, {})
-
-    targets = Targets(1, SPEC_VERSION, EXPIRY, {})
-    md_targets: Metadata[Targets] = Metadata(targets, {})
+    md_root = Metadata(Root(expires=EXPIRY))
```

Comment:
I think because of the use of default values for Metadata and Root attributes you can remove `ROLES` as well (line 69).

Reply:
right, same is likely true for SPEC_VERSION. 

Lint won't complain about unused variables because they're globals so they get exported as module variables. That's a bit annoying, I wonder if there's a best practice here...